src/providers/twitch/TwitchMessageBuilder.hpp

```diff
@@ -109,6 +110,7 @@ class TwitchMessageBuilder : public SharedMessageBuilder
     bool bitsStacked = false;
     bool historicalMessage_ = false;
     std::shared_ptr<MessageThread> thread_;
+    int messageOffset_ = 0;
```

Comment:
Comment for what its purpose is (e.g. offset to be used on index-based operations on originalMessage)